### ISSUE TITLE
bug within the analysis.py (not able to pass **kwargs)

### ISSUE THREAD

**Issue by zs-liu** [InstituteforDiseaseModeling/covasim#375]
I encountered this bug when I tried to pass **kwargs to `Sim.compute_fit(self, *args, **kwargs):` (so that I could compute MSE)
```python
def compute_fit(self, *args, **kwargs):
    self.fit = cva.Fit(self, *args, **kwargs)
    return self.fit
```
Then I took a look at cva.Fit(self, *args, **kwargs):
```python
class Fit(Analyzer):
    def __init__(self, sim, weights=None, keys=None, custom=None, compute=True, verbose=False, die=True, **kwargs):
        super().__init__(**kwargs) # Initialize the Analyzer object
        # something else here
```

However, the `Analyzer.__init__()` doesn't accept any **kwargs.
```python
class Analyzer(sc.prettyobj):
    def __init__(self, label=None):
    # something else here
```

**Comment by cliffckerr** [comment]
Well spotted, looks like a bug!

**Comment by cliffckerr** [comment]
Fixed in the 3.1.0 release coming soon.
